feature-matcher-generator-e2e-test/src/main/java/ru/yandex/qatools/proc/test/Lord.java

```diff
@@ -8,7 +9,7 @@
 
 @GenerateMatcher
 public class Lord {
-
+    @DoNotGenerateMatcher
```

Comment:
what will be if we add both this and other annotation? I think it should be preferred. Can you add a test case for this situation?